src/Form/TaskExportType.php

```diff
@@ -46,66 +50,127 @@ public function buildForm(FormBuilderInterface $builder, array $options)
+                        "TasksExportUnified.taskBeforeDay",
+                        "TasksExportUnified.taskBeforeTime",
+                        "TasksExportUnified.taskStatus",
+                        "TasksExportUnified.taskComments",
+                        "TasksExportUnified.taskDoneNotes",
+                        "TasksExportUnified.taskFailedNotes",
+                        "TasksExportUnified.taskFinishedAtDay",
+                        "TasksExportUnified.taskFinishedAtTime",
+                        "TasksExportUnified.taskCourier",
+                        "TasksExportUnified.taskTags",
+                        "TasksExportUnified.addressContactName",
+                        "TasksExportUnified.taskOrganizationName",
+                        "TasksExportUnified.taskPosition"
+                    ]
+                  ]
+                ])
+            ]);
+
+            // Need to invoke a method on the Response,
+            // to actually throw the Exception here
+            // https://github.com/symfony/symfony/issues/34281
+            // https://symfony.com/doc/5.4/http_client.html#handling-exceptions
+            $content = $response->getContent();
+
+            $resultSet = json_decode($content, true);
 
             $csv = CsvWriter::createFromString('');
             $csv->insertOne([
                 '#',
+                '# order',
+                'orderNumber',
                 'type',
                 'address.name',
                 'address.streetAddress',
                 'address.latlng',
                 'address.description',
-                'after',
-                'before',
+                'afterDay',
+                'afterTime',
+                'beforeDay',
+                'beforeTime',
                 'status',
                 'comments',
                 'event.DONE.notes',
                 'event.FAILED.notes',
-                'finishedAt',
+                'finishedAtDay',
+                'finishedAtTime',
                 'courier',
                 'tags',
                 'address.contactName',
                 'organization'
             ]);
 
             $records = [];
-            foreach ($tasks as $task) {
-
-                $address = $task->getAddress();
-                $finishedAt = '';
+            foreach ($resultSet['data'] as $resultObject) {
 
-                if ($task->hasEvent(Event\TaskDone::messageName())) {
-                    $finishedAt = $task
-                        ->getLastEvent(Event\TaskDone::messageName())
-                        ->getCreatedAt()->format('Y-m-d H:i:s');
-                } else if ($task->hasEvent(Event\TaskFailed::messageName())) {
-                    $finishedAt = $task
-                        ->getLastEvent(Event\TaskFailed::messageName())
-                        ->getCreatedAt()->format('Y-m-d H:i:s');
-                }
+                $address = $this->geocoder->geocode($resultObject['TasksExportUnified.addressStreetAddress']);
```

Comment:
We already have the coordinates in the `geo` field, we shouldn't need to use the geocoder for this.